bot/extensions/threads_cog.py

```diff
@@ -120,6 +133,47 @@ def cog_load(self):
             )
         )
 
+        # Runs reminders everyday at 12:30
+        self.jobs.append(
+            self.bot.scheduler.add_job(
+                self.daily_reminder, "cron", hour=12, minute=30, timezone=self.timezone
+            )
+        )
+
+    async def daily_reminder(self):
+        """Send a daily reminder for active threads."""
+        info("Posting daily threads's reminder")
+
+        embed = Embed(
+            color=self.bot.default_color,
+            title="🔔 Daily Reminder",
+            description="Join the discussion in the latest active threads:",
+        )
+
+        if threads := Thread.all():
+            for thread in threads:
+                if (
+                    hasattr(thread, "latest_thread")
+                    and thread.latest_thread
+                    and thread.daily_reminder
+                ):
+                    discord_thread = await self.bot.fetch_channel(
+                        int(thread.latest_thread)
+                    )
+                    if getattr(discord_thread, "archived", False) or getattr(
```

Comment:
I'm assuming you're doing this because achived return None? If it's the case, you don't need to do it. You can use `discord_thread.archived` directly; None is evaluated to false.